### ISSUE TITLE
Web - Secondary login added in OldDot is can't be seen in NewDot

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#49489]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v9.0.38-3
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Sign in to staging.new.expensify.com on Primary device
2. Sign in to staging.expensify.com with a secondary device with same account as step 1
3. Add and validate a secondary login with staging.expensify.com
4. On the primary device navigate to Profile > Contact Method

## Expected Result:
Secondary login set in OldDot should be visible on NewDot

## Actual Result:
Secondary login set in OldDot is not visible on NewDot

## Workaround:
Unknown

## Platforms:

- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/a07b3802-65d3-4008-a5df-e2970d45fb7e

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by IuliiaHerets** [comment]
@puneetlath  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by puneetlath** [comment]
This is a backwards compatibility thing. We don't send a pusher update when this is added from OldDot. We probably should, but it's not very high priority. I'm going to put it in #wave-control.

**Comment by bfitzexpensify** [comment]
I think `weekly` remains the right priority for this - labelled and ready to be picked up as a hot pick

**Comment by bfitzexpensify** [comment]
Same update as above

**Comment by bfitzexpensify** [comment]
Still waiting on a volunteer, remains low priority

**Comment by bfitzexpensify** [comment]
Same update, waiting on a volunteer

**Comment by bfitzexpensify** [comment]
Still waiting on a volunteer

**Comment by bfitzexpensify** [comment]
Same update - still waiting on a volunteer

**Comment by bfitzexpensify** [comment]
Same update - still waiting on a volunteer

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by kavimuru** [comment]
Bug is not reproducible

https://github.com/user-attachments/assets/3e1abb18-07cf-42a8-b3e3-bd437893541f
